docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst

```diff
@@ -0,0 +1,53 @@
+My Journey to GSoC 2022
+===============
```

Comment:
too short, should be the same size as the title. the doc generation creates multiple warnings.

Reply:
Hi @skoudoro , I tried generating the blog post. I got a warning while generating the doc
`/home/shivam/fury/docs/source/posts/2022/2022-05-24-my-journey-to-gsoc-2022-shivam.rst: WARNING: document isn't included in any toctree` . But the generated doc was fine.
![Screenshot from 2022-05-28 09-46-04](https://user-images.githubusercontent.com/74976752/170809640-d29ce9f0-d882-4b2e-b68c-6c8777856823.png)